client/src/Auth/Register/Register.js

```diff
@@ -0,0 +1,89 @@
+import React from 'react';
+// https://www.flaticon.com/free-icon/circular-avatar_108331
+import avatar from './avatar.png';
+import './Register.css';
+
+const USERNAME_LENGTH_MIN = 5
+const PASSWORD_LENGTH_MIN = 6
+
+// Checks whether state values meets valid criteria
+function validateInputs ({ username, password, confirmPassword }) {
+    
+    return {
+        username: username.length < USERNAME_LENGTH_MIN,
+        password: password.length < PASSWORD_LENGTH_MIN,
+        confirmPassword: confirmPassword !== password
+    }
+}
+
+// If an input is not focused has value but not complete returns false
+function checkDirty (focus, {username, password, confirmPassword}, errors) {
+
+    return {
+        username: focus !== 'username' && errors.username && username.length > 0,
+        password: focus !== 'password' && errors.password && password.length > 0,
+        confirmPassword: focus !== 'confirmPassword' && errors.confirmPassword && password.length > 0
+    }
+}
+
+const Register = (props) => {
+
+    const { focus } = props.inputs;
+    const errors = validateInputs(props.inputs);
+    const isDirty = checkDirty(focus, props.inputs, errors);
+    const isEnabled = Object.keys(errors).some(e => errors[e]);
+
+    return (
+        <div className="container-fluid d-flex flex-column my-auto">
+                <div className="d-flex flex-column col-xl-4 col-lg-5 col-md-6 col-sm-8 col-9 mx-auto">
+                    <div className="align-center pt-2 mb-3">
+                        <h2 className="register-text">Register</h2>
+                    </div>
+                    <img src={avatar} className="align-self-center my-3" width="120px" />
```

Comment:
Please add an alt text to make the ESLint warning disappear. I think `alt=""` will suffice since the image is just decorative.